documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -135,7 +135,7 @@ spec:
 EOF
 ----
 <1> Specify the name of the VMware `Provider` CR.
-<2> Specify the managed object reference (MOR) of the VMware host.
+<2> Specify the managed object reference (MOR) of the VMware host. To retrieve the MOR, see xref:retrieving-vmware-mor_{context}[Retrieving a VMware vSphere MOR].
```

Comment:
This xref is wrong. Use the actual context, not the context attribute. It will generate a build error when we move it to Pantheon

Reply:
WRONG - apologies